radicle-crdt/src/lwwset.rs

```diff
@@ -0,0 +1,145 @@
+use crate::{lwwmap::LWWMap, Semilattice};
+
+/// Last-Write-Wins Set.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct LWWSet<T, C> {
+    inner: LWWMap<T, (), C>,
```

Comment:
I'm not sure that's possible, because two values in a `BTreeSet` are either equal, in which case it's the same value, or they would occupy different slots, in which case you wouldn't know that they should be merged. See what I mean?

Reply:
Put differently, `Semilattice` for `BTreeSet` would be a "GSet", ie. grow-only-set, which is not super useful here.

Reply:
Ah, I didn't think about that enough :) Thanks for the explanation :+1: